richeditor-compose/src/commonMain/kotlin/com/mohamedrejeb/richeditor/model/RichTextState.kt

```diff
@@ -186,13 +186,15 @@ class RichTextState internal constructor(
         codeColor: Color = Color.Unspecified,
         codeBackgroundColor: Color = Color.Unspecified,
         codeStrokeColor: Color = Color.Unspecified,
+        listIndent: Int = 38
     ) {
         richTextConfig = RichTextConfig(
             linkColor = if (linkColor.isSpecified) linkColor else richTextConfig.linkColor,
             linkTextDecoration = linkTextDecoration ?: richTextConfig.linkTextDecoration,
             codeColor = if (codeColor.isSpecified) codeColor else richTextConfig.codeColor,
             codeBackgroundColor = if (codeBackgroundColor.isSpecified) codeBackgroundColor else richTextConfig.codeBackgroundColor,
             codeStrokeColor = if (codeStrokeColor.isSpecified) codeStrokeColor else richTextConfig.codeStrokeColor,
+            listIndent = listIndent
```

Comment:
Here we need to keep the same value if the `listIndent` is not changed.
So it's going to be something like this:
`if (listIndent > -1) listIndent else richTextConfig .listIndent`